components/clp-py-utils/clp_py_utils/initialize-spider-db.py

```diff
@@ -0,0 +1,301 @@
+    PRIMARY KEY (`id`)
+);
+""",
+    """
+CREATE TABLE IF NOT EXISTS `scheduler_leases`
+(
+    `scheduler_id` BINARY(16) NOT NULL,
+    `task_id`      BINARY(16) NOT NULL,
+    `lease_time`   TIMESTAMP  NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    CONSTRAINT `lease_scheduler_id` FOREIGN KEY (`scheduler_id`) REFERENCES `schedulers` (`id`) ON UPDATE NO ACTION ON DELETE CASCADE,
+    CONSTRAINT `lease_task_id` FOREIGN KEY (`task_id`) REFERENCES `tasks` (`id`) ON UPDATE NO ACTION ON DELETE CASCADE,
+    INDEX (`scheduler_id`),
+    PRIMARY KEY (`scheduler_id`, `task_id`)
+);
+""",
+    """
+CREATE TABLE IF NOT EXISTS `data_locality`
+(
+    `id`      BINARY(16)  NOT NULL,
+    `address` VARCHAR(40) NOT NULL,
+    KEY (`id`) USING BTREE,
+    CONSTRAINT `locality_data_id` FOREIGN KEY (`id`) REFERENCES `data` (`id`) ON UPDATE NO ACTION ON DELETE CASCADE
+);
+""",
+    """
+CREATE TABLE IF NOT EXISTS `data_ref_driver`
+(
+    `id`        BINARY(16) NOT NULL,
+    `driver_id` BINARY(16) NOT NULL,
+    KEY (`id`) USING BTREE,
+    KEY (`driver_id`) USING BTREE,
+    CONSTRAINT `data_driver_ref_id` FOREIGN KEY (`id`) REFERENCES `data` (`id`) ON UPDATE NO ACTION ON DELETE CASCADE,
+    CONSTRAINT `data_ref_driver_id` FOREIGN KEY (`driver_id`) REFERENCES `drivers` (`id`) ON UPDATE NO ACTION ON DELETE CASCADE
+);
+""",
+    """
+CREATE TABLE IF NOT EXISTS `data_ref_task`
+(
+    `id`      BINARY(16) NOT NULL,
+    `task_id` BINARY(16) NOT NULL,
+    KEY (`id`) USING BTREE,
+    KEY (`task_id`) USING BTREE,
+    CONSTRAINT `data_task_ref_id` FOREIGN KEY (`id`) REFERENCES `data` (`id`) ON UPDATE NO ACTION ON DELETE CASCADE,
+    CONSTRAINT `data_ref_task_id` FOREIGN KEY (`task_id`) REFERENCES `tasks` (`id`) ON UPDATE NO ACTION ON DELETE CASCADE
+);
+""",
+    """
+CREATE TABLE IF NOT EXISTS `client_kv_data`
+(
+    `kv_key`    VARCHAR(64)    NOT NULL,
+    `value`     BLOB(60000) NOT NULL,
+    `client_id` BINARY(16)     NOT NULL,
+    PRIMARY KEY (`client_id`, `kv_key`)
+);
+""",
+    """
+CREATE TABLE IF NOT EXISTS `task_kv_data`
+(
+    `kv_key`  VARCHAR(64)    NOT NULL,
+    `value`   BLOB(60000) NOT NULL,
+    `task_id` BINARY(16)     NOT NULL,
+    PRIMARY KEY (`task_id`, `kv_key`),
+    CONSTRAINT `kv_data_task_id` FOREIGN KEY (`task_id`) REFERENCES `tasks` (`id`) ON UPDATE NO ACTION ON DELETE CASCADE
+);
+""",
+]
+
+
+def main(argv):
+    args_parser = argparse.ArgumentParser(description="Sets up Spider database.")
+    args_parser.add_argument("--config", "-c", required=True, help="CLP configuration file.")
+    parsed_args = args_parser.parse_args(argv[1:])
+
+    config_path = pathlib.Path(parsed_args.config)
+    try:
+        clp_config = ClpConfig.model_validate(read_yaml_config_file(config_path))
+        clp_config.database.load_credentials_from_env()
+        clp_config.database.load_credentials_from_env(user_type=ClpDbUserType.CLP)
```

Comment:
since we load for CLP already
```suggestion
        clp_config.database.load_credentials_from_env(user_type=ClpDbUserType.CLP)
```